### PR TITLE
feat(glitter): centralize shared context and refresh

### PR DESCRIPTION
**Pull request by shepherdjerred**
## Summary

- Adds `@shepherdjerred/glitter-context`, a language-neutral canonical data package with JSON Schema plus Zod and Pydantic validation, and a browser/Node-safe built bundle with inlined data.
- Migrates Birmel, Scout, and Glitter to the shared people, lore, style-card, and current-relationship projections; removes the duplicate canonical copies.
- Adds the weekly Temporal refresh workflow on its own single-concurrency queue, with checksum-verified corpus reads, deterministic candidate selection, GPT-5.5 structured generation, evidence-bound relationship updates, scoped human-review PRs, replay coverage, rehearsal, metrics, and dashboard panels.

This is the second and final layer of the stack, on top of #1693.

## Relationship history

Relationships are append-only events with explicit current/historical state and `supersedes` provenance. Caitlyn and Richard retain their historical Dating event while the current projection exposes Exes.

## Safety and determinism

- Reads only a fully verified snapshot mirrored across SeaweedFS and R2.
- Refreshes a style card after 20 new safe messages or 90 days, using a bounded deterministic sample.
- Requires high-confidence relationship proposals with 2–8 exact message IDs and first-party evidence.
- Rejects edits outside the approved shared-context files and emits no PR for an unchanged result.
- Creates human-review PRs only; it never auto-merges.
- Registers the Monday schedule paused until credentialed acceptance is complete.

## Verification

- `bun run verify -- --affected`: 76/76 tasks passed; the exact staged commit gate then passed 71/71 tasks.
- Temporal: 667 tests passing, including workflow replay and relationship/watermark edge cases.
- Scout: 1,185 backend tests and 494 data tests passing.
- Shared package: TypeScript schema/consumer tests and Pydantic validation over 71 people, 80 relationship events, and 13 style cards.
- Temporal worker Docker smoke image passed.
- Clean-clone rehearsal passed deterministic generation, idempotence, and approved-path scope.
- Glitter was built and inspected through PinchTab; the rendered graph shows Caitlyn/Richard as Exes.

Glitter relationship graph rendered from the shared current projection

## Live acceptance still required

This PR intentionally remains draft. Discord/R2 credentials were not supplied and local 1Password is not signed in, so credentialed Discord, object-storage, Temporal, OpenAI, generated-PR, and live Birmel acceptance have not run. The full historical scrape remains separately gated on explicit approval of the Discord public-channel/thread inventory in #1693.

### CODE REVIEW

**Comment by shepherdjerred** [comment]
This change is part of the following stack:

- #1700 ◀

<sub>Change managed by [git-spice](https://abhinav.github.io/git-spice/).</sub>

**Comment by shepherdjerred** [comment]
@codex review

**Comment by shepherdjerred** [comment]
@codex review

**Comment by shepherdjerred** [comment]
@codex review

File: packages/birmel/src/elections/candidates.ts [hunk 1]
```diff
@@ -1,20 +1,11 @@
...
 
-export async function getAllCandidates(): Promise<string[]> {
-  const styleCardsDir = path.join(import.meta.dir, "../persona/style-cards");
-  const files = await readdir(styleCardsDir);
-
-  return files
-    .filter((f) => f.endsWith("_style.json"))
-    .map((f) => f.replace("_style.json", ""));
+export function getAllCandidates(): string[] {
+  return listStyleCardNames();
```
[the remark this answers is not in the document]
Comment (shepherdjerred) [reply packages/birmel/src/elections/candidates.ts:4]: Addressed in 3194b023d: `getDiscordIdForPersona` now derives the profile id from the shared glitter-context person data (`getPerson(persona)?.discordUserIds[0]`) instead of a hardcoded 10-persona map, so every `listStyleCardNames()` candidate (including Caitlyn/Colin/Richard) resolves a profile id automatically.

File: packages/temporal/src/activities/glitter-context-refresh-paths.ts [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+export const GLITTER_PEOPLE_PATH = `${PACKAGE_PATH}/data/people.json`;
+export const GLITTER_RELATIONSHIPS_PATH = `${PACKAGE_PATH}/data/relationships.json`;
+export const GLITTER_GENERATED_DATA_PATH = `${PACKAGE_PATH}/src/generated-data.ts`;
+
+export function isAllowedGlitterContextRefreshPath(path: string): boolean {
+  return (
+    path === GLITTER_GENERATION_STATE_PATH ||
+    path === GLITTER_RELATIONSHIPS_PATH ||
+    path === GLITTER_GENERATED_DATA_PATH ||
+    /^packages\/glitter-context\/data\/style-cards\/[a-z0-9-]+_style\.json$/u.test(
```
[the remark this answers is not in the document]
Comment (shepherdjerred) [reply packages/temporal/src/activities/glitter-context-refresh-paths.ts:13]: Addressed in 3194b023d: added `packages/glitter-context` to `site_glitter_paths` and `site_scout_paths` in `.buildkite/scripts/ci-changed.sh` (the `sites` aggregate inherits it via the union). The `images` lane already routes glitter-context changes to the birmel/scout/temporal images via the workspace dependency closure in `select-image-targets.ts` — verified empirically that a glitter-context data change selects `["birmel","scout-for-lol","temporal-worker"]`, so no change was needed there.

File: packages/temporal/src/schedules/register-schedules.ts [hunk 3]
```diff
@@ -349,14 +350,28 @@ export const SCHEDULES: ScheduleDefinition[] = [
...
+    cronExpression: "0 11 * * 1",
+    taskQueue: TASK_QUEUES.GLITTER_CONTEXT,
+    overlap: ScheduleOverlapPolicy.SKIP,
+    workflowExecutionTimeout: "3 hours",
+    memo: "Weekly GPT-5.6 Sol refresh of shared Glitter style cards and evidence-backed relationship history from the verified mirrored corpus",
+    initialPauseNote:
+      "Awaiting credentialed dry-run against the first approved complete snapshot",
+    requiredEnvironment: [
+      "GLITTER_DISCORD_GUILD_ID",
+      ...GLITTER_CORPUS_STORAGE_ENV,
```
[the remark this answers is not in the document]
Comment (shepherdjerred) [reply packages/temporal/src/schedules/register-schedules.ts:370]: The schedules already fail-safe: `schedule-state.ts` registers both paused while any required env var is absent. Fully wiring the 12 `GLITTER_*` credentials into `worker.ts` is operator-blocked — the fields do not exist in the `temporal-temporal-worker-1p` 1Password item, so required secret refs fail `check:1password` (verified), and the repo forbids `optional: true` secrets. Populating them needs the real Discord + S3/R2 credentials plus a vault-snapshot refresh, all operator-only. Tracked in `packages/docs/todos/glitter-corpus-worker-credentials.md`.

File: packages/glitter-context/src/schema.ts [hunk 4]
```diff
@@ -0,0 +1,194 @@
...
+    $schema: z.string().optional(),
+    schemaVersion: z.literal(1),
+    people: z.array(PersonSchema),
+  })
+  .superRefine((document, context) => {
+    const ids = new Set<string>();
+    const aliases = new Set<string>();
+    for (const person of document.people) {
+      if (ids.has(person.id)) {
+        context.addIssue({
```
[the remark this answers is not in the document]
Comment (shepherdjerred) [reply packages/glitter-context/src/schema.ts:33]: Addressed in 3194b023d: the people validator's `superRefine` now tracks `discordUserId`s globally and raises a `duplicate Discord user id` issue, rejecting the same id assigned to multiple records. The Python validator mirrors the same check.

File: packages/birmel/package.json [hunk 5]
```diff
@@ -45,6 +45,7 @@
     "@prisma/client-runtime-utils": "7.8.0",
     "@sentry/bun": "10.53.1",
     "@shepherdjerred/llm-observability": "workspace:*",
+    "@shepherdjerred/glitter-context": "workspace:*",
```
[the remark this answers is not in the document]
Comment (shepherdjerred) [reply packages/birmel/package.json:49]: Addressed in 3194b023d: `dev` and `start` now run `bun run build:context` (`turbo run build --filter=@shepherdjerred/glitter-context`) first, so the gitignored `dist/` exists before Birmel's source imports it. (The Docker image already builds glitter-context in its build stage and runs `bun src/index.ts` directly, so only the local launch path needed this.)
